src/accuracy/tests/analytical_variance_for_sampling_liquid_legions_test.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2021 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for analytical_variance_for_sampling_liquid_legions.py."""
+
+
+from numpy.lib.polynomial import RankWarning
+from absl.testing import absltest
+from absl.testing import parameterized
+import numpy as np
+
+from wfa_planning_evaluation_framework.accuracy.analytical_variance_for_sampling_liquid_legions import (
+    AnalyticalAccuracyEvaluator,
+)
+from wfa_cardinality_estimation_evaluation_framework.estimators.same_key_aggregator import (
+    ExponentialSameKeyAggregator,
+    StandardizedHistogramEstimator,
+)
+from wfa_cardinality_estimation_evaluation_framework.estimators.estimator_noisers import (
+    LaplaceEstimateNoiser,
+)
+
+reach_epsilon = 0.1  # dp epsilon for reach
```

Comment:
Consider moving this code into a setUpClass() method.

https://docs.python.org/3/library/unittest.html#setupclass-and-teardownclass